healthcareai/common/model_eval.py

```diff
@@ -175,14 +304,21 @@ def roc_plot_from_thresholds(roc_thresholds_by_model, save=False, debug=False):
         best_true_positive_rate = metrics['best_true_positive_rate']
         best_false_positive_rate = metrics['best_false_positive_rate']
 
+        # Build the legend for each line including positive label if specified
+        try:
+            positive_label = metrics['positive_label']
+            label = '{} (PR AUC = {}, positive label: {})'.format(model_name, round(roc_auc, 2), positive_label)
```

Comment:
This should be ROC AUC. Similarly on line 312.

Reply:
Nice catch!